src/panels/config/integrations/integration-panels/matter/dialog-matter-ping-node.ts

```diff
@@ -75,7 +85,10 @@ class DialogMatterPingNode extends LitElement {
                 <mwc-list>
                   ${this._pingResultEntries.map(
                     ([ip, success]) =>
-                      html`<ha-list-item hasMeta noninteractive
+                      html`<ha-list-item
+                        hasMeta
+                        .ip=${ip}
+                        @click=${this._copyIpToClipboard}
```

Comment:
UX wise it is very unclear that we are going to copy the IP when you click the item, ideally we would have a button with `mdiContentCopy` somewhere.